core/templates/dev/head/pages/exploration_editor/feedback_tab/FeedbackTab.js

```diff
@@ -66,6 +67,13 @@ oppia.controller('FeedbackTab', [
     $q.all([userInfoPromise, threadPromise]).then(function() {
       $rootScope.loadingMessage = '';
     });
+    $scope.fetchThreadsAgain = function() {
```

Comment:
The name is misleading since it only fetches threads if an update was detected.

Better to have the inner part be "fetchUpdatedThreads". The "onBackButtonClicked" function would then contain the if statement logic, and call fetchUpdatedThreads() within it. If there's any potential for this to be unclear, consider adding comments to explain.

In general please be careful about naming things really clearly. The underlying conceptual underpinnings need to be correct otherwise future devs (who don't know anything about the discussion in this PR) are going to have a hard time understanding what is going on.

Reply:
Okay @seanlip , will take care of this.